### DISCUSSION TITLE
Algebraic linear system

### DISCUSSION THREAD

**Discussion by riccardotomada** [lululxvi/deepxde#943] [Q&A]
Hi all,

I would like to ask if it is possible to use as a loss function the residual of a system of algebraic equations in DeepXDE, since I would like to develop a reduced order model for a parameterized PDE.
Basically I have an array of 10 entries which corresponds to 10 samples of a parameter in range [0,1].
I have obtained via FEM method a full order model in the form of Ay = b. I then projected A and b on a low rank linear space obtained via singular value decomposition.
Now I have the ROM system in the form A'y'=b', where the matrix A' shape is [15,15] and it is the same for each parameter value, whereas the rhs term b' is everytime different. I stored the b' arrays in a matrix B of shape [15,10]. I would like the network to learn the map between the parameter space and the reduced coefficients y', which for each parameter value are an array of shape [15,1].

Is it feasible in DeepXDE? I did some attempts but it seems that initial conditions are always required. 

Thanks in advance to anyone. Have a nice weekend.

**Comment by lululxvi** [comment]
Not fully understand your question, but IC is not always required.
